app/queries/create_course_export_mutator.rb

```diff
@@ -5,6 +5,7 @@ class CreateCourseExportMutator < ApplicationQuery
   property :course_id, validates: { presence: { message: 'CourseIdBlank' } }
```

Comment:
If the updates involve Graph mutations ensure that the files are migrated to the new approach without a mutator.
 
 Checkout `app/graphql/mutations/create_course.rb` for inspiration.